rasa_core/trackers.py

```diff
@@ -145,12 +145,13 @@ def past_states(self, domain: 'Domain') -> deque:
         generated_states = domain.states_for_tracker_history(self)
         return deque((frozenset(s.items()) for s in generated_states))
 
-    def change_form_to(self, form_name: Text) -> None:
+    def change_form_to(self, form_name: Text, trigger_message: Text) -> None:
```

Comment:
Actually I think adding trigger message to the `Form` event is unnecessary. At this point `'trigger_message': self.latest_message`, no?